plugin/webhook/webhook.go

```diff
@@ -34,17 +34,66 @@ const (
 	WebhookError Level = "ERROR"
 )
 
+// EventType is eventType of webhook.
+type EventType string
+
+const (
+	IssueCreated                   EventType = "issue_created"
+	IssueUpdated                   EventType = "issue_updated"
+	IssueStatusUpdated             EventType = "issue_status_updated"
+	IssueCommentCreated            EventType = "issue_comment_created"
+	IssuePipelineTaskStatusUpdated EventType = "issue_pipeline_task_status_updated"
+)
+
+// ObjectKind kind of object. for now, only Issue.
+type ObjectKind string
+
+// Issue object kind
+const Issue ObjectKind = "issue"
+
+// IssueObject object of issue
+type IssueObject struct {
+	ID          int
+	Name        string
+	Status      string
+	Type        string
+	Description string
+	AssigneeID  int
```

Comment:
Besides ID, name and email are more useful since the ID is likely not that useful to the external system.

Reply:
If we can ensure that Name can be used as a unique identifier, it makes sense to use only Name